docs/metrics/README.md

```diff
@@ -417,6 +417,73 @@ SDK to reclaim unused metric points.
   Specification](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#overflow-attribute)
   become stable, this feature will be turned on by default.
 
+#### Cardinality overflow example
+
+In the following example we set the maximum number of `MetricPoint`s allowed to
+be `3` for two metrics. This means that for these metrics, the SDK will only
+export measurements for up to `3` distinct key/value combinations.
+
+> [!NOTE]
+> One `MetricPoint` is reserved on every `Metric` for the special case where
```

Comment:
I think we should also explicitly mention that cardinality is used to determine the number of MetricPoints allocated for the metric to establish the relationship between cardinality and MetricPoint.

Reply:
I updated to avoid mentioning `MetricPoint`: https://github.com/open-telemetry/opentelemetry-dotnet/pull/5349/commits/f6014118bca0d22ada33eb4ed20d4476c8af3666

Hoping to not introduce another concept into an already very complicated topic. May or may not have succeeded 🤣